src/patternfly/demos/Page/examples/Page.css

```diff
@@ -0,0 +1,6 @@
+#ws-html-demos-c-page-context-selector-expanded-in-sidebar .pf-v6-c-page__sidebar-body.pf-m-context-selector .pf-v6-c-menu {
+  position: absolute;
+  left: var(--pf-v6-c-page__sidebar-body--m-context-selector--PaddingInlineStart);
+  right: var(--pf-v6-c-page__sidebar-body--m-context-selector--PaddingInlineEnd);
+  z-index: var(--pf-t--global--z-index--sm);
```

Comment:
Gonna leave some nit comments, not necessary for merging or anything. I might add `#ws-html-demos-c-page-context-selector-expanded-in-sidebar .pf-v6-c-page__sidebar-body.pf-m-context-selector { position: relative; }` then update this and add a `top: 100%` so it's positioned from the sidebar-body element. Right now I think it's positioned via the parent `.pf-c-sidebar` element.